Rakefile

```diff
@@ -47,6 +47,14 @@ def search_server
   search_config.search_server
 end
 
+def bucket
```

Comment:
I'm not sure but could we have `Snapshot::SnapshotRepository` access `SearchConfig.new.elasticsearch["base_uri"]` directly? That would stop us from having to use `elasticsearch_uri` in the rake tasks/initialiser altogether.